domains/certificates/Query.API/API.UnitTests/DataSyncSyncer/DataSyncServiceTest.cs

```diff
@@ -88,7 +88,7 @@ public async Task FetchMeasurements_MeteringPointOnboarded_NoDataFetched()
         var response = await service.FetchMeasurements(masterData,
             CancellationToken.None);
 
-        Assert.Empty(response);
+        response.Should().BeEmpty();
```

Comment:
Should we agree on always using FluentAssertions in our tests?

Reply:
I would like that :) 
I prefer FluentAssertions as they are easier to read and makes testing for equality between objects easy